src/routes/safe/components/Apps/utils.js

```diff
@@ -15,22 +18,44 @@ export const getAppInfoFromOrigin = (origin: string) => {
 }
 
 export const getAppInfoFromUrl = async (appUrl: string) => {
+  let res = { id: undefined, url: appUrl, name: 'unknown', iconUrl: appsIconSvg, error: true }
+
   try {
     const appInfo = await axios.get(`${appUrl}/manifest.json`)
```

Comment:
`appUrl` should probably be cleaned (trim whitespace, check for trailing `/`, stuff like that)